luaui/Widgets/gui_options.lua

```diff
@@ -2879,7 +2879,7 @@ function init()
 
 		{ id = "keybindings", group = "control", category = types.basic, name = Spring.I18N('ui.settings.option.keybindings'), type = "select", options = keyLayouts.keybindingLayouts, value = 1, description = Spring.I18N('ui.settings.option.keybindings_descr'),
 		  onload = function()
-			  local keyFile = Spring.GetConfigString("KeybindingFile", keyLayouts.keybindingPresets["Default"])
+			  local keyFile = Spring.GetConfigString("KeybindingFile", keyLayouts.keybindingLayoutFiles[1])
```

Comment:
I'd prefer if we used the explicit denomination here like previous, instead of going by index, e.g. `keyLayouts.keybindingPresets["Grid Optimized"]`

Reply:
My thought with this specifically is that any change to the name of the presets, or any future change to the default preset will result in this breaking if all the appropriate strings are not updated. Using the indexed value means that the first item in that preset list will be treated as default, regardless of name. Granted, the flip side of that is that someone could rearrange that list without realizing this is keyed to it could break things, but I feel like that's a less likely outcome. I'm open to having my mind changed on that though.